openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -16,18 +20,66 @@
     create_simple_signal,
 )
 from openedx_events.tests.utils import FreezeSignalCacheMixin
-from openedx_events.tooling import OpenEdxPublicSignal, load_all_signals
+from openedx_events.tooling import KNOWN_UNSERIALIZABLE_SIGNALS, OpenEdxPublicSignal, load_all_signals
 
-# If a signal is explicitly not for use with the event bus, add it to this list
-#  and document why in the event's annotations
-KNOWN_UNSERIALIZABLE_SIGNALS = [
-    "org.openedx.learning.discussions.configuration.changed.v1",
-    "org.openedx.content_authoring.course.certificate_config.changed.v1",
-    "org.openedx.content_authoring.course.certificate_config.deleted.v1",
-]
+
+def generate_test_data_for_schema(schema):  # pragma: no cover
```

Comment:
Curious why `# pragma: no cover` is needed for this and the other method, when it seems like they are used in tests?

Reply:
I see. You could add the pragma more specifically on the code that isn't actually covered, but this is non-blocking. It just raised questions for me. Thanks.

Reply:
I thought it worthwhile to future-proof. Coverage shouldn't really care about this method at all, and if someone changes it they shouldn't have to go through the annoyance of figuring out which bits are and aren't covered again.